economy/static/economy/css/economy_common.css

```diff
@@ -16,8 +16,38 @@
     color: #103242;
 }
 
-.economy__leftbutton.active{
+.economy__leftbutton.active {
     font-size: 24px;
     font-weight: 600;
     text-transform: uppercase;
+}
+
+.main-table {
```

Comment:
It's slightly weird to name a flex-container "table".

If this is to be a tabular format, we should probably use a table element or a grid.

Reply:
It seems the flex views also do not align. See

![2020-12-03-181048_916x154_scrot](https://user-images.githubusercontent.com/3416694/101063745-78ea0780-3593-11eb-96da-53aa687055ec.png)